app/import_functions.go

```diff
@@ -982,8 +989,9 @@ func (a *App) importReaction(data *ReactionImportData, post *model.Post, dryRun
 	}
 
 	var user *model.User
-	user, err = a.Srv().Store.User().GetByUsername(*data.User)
-	if err != nil {
+	var nErr error
+	user, nErr = a.Srv().Store.User().GetByUsername(*data.User)
+	if nErr != nil {
 		return model.NewAppError("BulkImport", "app.import.import_post.user_not_found.error", map[string]interface{}{"Username": data.User}, err.Error(), http.StatusBadRequest)
```

Comment:
`nErr.Error()`

Actually there's no reason to declare `err` in L986. We can do `:=` inside the if condition itself.

Reply:
Fixed